pkg/config/testdata/TestLoadConfigSuccess/config.golden.yaml

```diff
@@ -331,6 +331,20 @@ sources:
                 enabled: true
                 config:
                     field: value
+                context:
+                    kubeconfig:
+                        defaultnamespace: ""
```

Comment:
🤔 It should be camel case. Can you add the yaml struct tags for the config to make sure all properties are marshalled with proper names? Thanks!